packages/react-components/scripts/build.js

```diff
@@ -52,6 +52,7 @@ const fileNames = [].concat.apply(
       ignore: ['./**/*.unit.spec.@(js|jsx)', './**/*.stories.@(js|jsx)'],
     }),
     glob.sync('./src/helpers/*.js'),
+    glob.sync('../core/src/i18n/**/*.js'),
```

Comment:
The build for `react-components` doesn't give us a proper bundle - it's just a bunch of JS files at the root level with all of their imports flattened. This makes sure that the "built" files will have access to the shared i18n setup.